src/socketio.quiver.js

```diff
@@ -1197,31 +1204,20 @@ QuiverSocialProvider.prototype.emitEncrypted_ = function(connections, friend,
  * @override
  */
 QuiverSocialProvider.prototype.logout = function(continuation) {
-  if (!this.isOnline_()) { // We may not have been logged in
-    continuation(undefined, this.err("OFFLINE"));
-    return;
-  }
-
-  var onClose = function(serverKey, continuation) {
-    this.log_('Deleting server ' + serverKey);
-    delete this.connections_[serverKey];
-    if (!this.isOnline_()) {
-      continuation();
-    }
-  };
+  this.dispatchEvent('onClientState', {
```

Comment:
I think it would be better to move this after the cleanup loop, and to call `this.makeClientState_` instead of constructing the event as a literal.